src/main/java/games/strategy/triplea/ui/menubar/LobbyMenu.java

```diff
@@ -161,7 +162,7 @@ private void addBanUsernameMenu(final JMenu parentMenu) {
       final IModeratorController controller = (IModeratorController) lobbyFrame.getLobbyClient().getMessengers()
           .getRemoteMessenger().getRemote(ModeratorController.getModeratorControllerName());
       try {
-        controller.banUsername(new Node(name1, InetAddress.getByName("0.0.0.0"), 0), Date.from(expire));
+        controller.banUsername(new Node(name1, InetAddress.getByName("0.0.0.0"), 0), Util.toRealDate(expire));
```

Comment:
Do any of the six calls to `Date#from()` in `LobbyFrame` need to be replaced with `Util#toRealDate()`, as well?  None of them explicitly add `Long.MAX_VALUE` milliseconds to the current instant, but the value does come from the user who could presumably just enter a really big number that would trigger an overflow.

Reply:
In this case we should probably limit the maximum allowed number in the entry field/use a JSpinner instead of a input field